### ISSUE TITLE
Download the pretrained model need permission ?

### ISSUE THREAD

**Issue by miaoYuanyuan** [soomean/Robust-Motion-Inbetweening-Reimplementation#1]

**Comment by soomean** [comment]
I changed the permission.
Please check.

**Comment by miaoYuanyuan** [comment]
> I changed the permission. Please check.
It can be accessed now. Thank you!

**Comment by miaoYuanyuan** [comment]
> I changed the permission. Please check.

It can be accessed now. Thank you!

**Comment by miaoYuanyuan** [comment]
@soomean is there any t-pose int the files? or where could I find the t-pose file?

**Comment by soomean** [comment]
@miaoYuanyuan Do you mean rest.bvh for lafan1 dataset?

**Comment by miaoYuanyuan** [comment]
yes, it used in save BVH, but I solved with another BVH with same skeleton. 
I have another question. I test this on pytorch1.7.0, but I find the result have illed pose in last frame or first some frames. is it normal? or I must use the pytorch1.9.0?

**Comment by soomean** [comment]
I don't think pytorch version is a big matter.
The results could be not so good in idle motions.
Please pull some changes that I just made and tell me if there's the same problem with other motions.

**Comment by miaoYuanyuan** [comment]
I change the test indices [100,500,1000,5000], but I find the groundtruth and predition result both are foot sliding?  is the reason of reduce mean of xz in function get_lafan1_set()? how to solve it ?

**Comment by miaoYuanyuan** [comment]
ok, I found the groundtruth foot sliding is due to the code " positions *= 0.15 ". but the  prediction result is foot sliding raw.

**Comment by soomean** [comment]
No, positions *= 0.15 is just for scaling for visualization.
the results should look like these (blue for gt and red for prediction):

![gt_100](https://user-images.githubusercontent.com/23622580/140469802-69a76bd6-676a-4f4f-b595-06cc28586d59.gif)
![prediction_100](https://user-images.githubusercontent.com/23622580/140469807-b1448821-aa49-490c-bd19-874a508508d0.gif)

Foot sliding can be shown in prediction results and it can be improved if you add a foot sliding loss.

**Comment by soomean** [comment]
I will close this one as the code doesn't seem to have any other issues for visualization.
Please make a new one if you need.
